configs/iri-pod.j2

```diff
@@ -22,9 +22,11 @@ spec:
         - "--snapshot"
         - "/iri/data/snapshot.txt"
         - "--testnet"
+        - "true"
         - "--testnet-coordinator"
         - "BTCAAFIH9CJIVIMWFMIHKFNWRTLJRKSTMRCVRE9CIP9AEDTOULVFRHQZT9QAQBZXXAZGBNMVOOKTKAXTB"
         - "--testnet-no-coo-validation"
+        - "true"
```

Comment:
I think it will be nicer if the arg will be in the same line as the config

Reply:
That would be not according to the other arguments/format

Reply:
:+1: